android/keyframes/src/main/java/com/facebook/keyframes/KeyframesDrawable.java

```diff
@@ -317,6 +333,11 @@ public void setFrameProgress(float frameProgress) {
     }
   }
 
+  public void seekToProgress(float progress) {
+    stopAnimation();
+    onProgressUpdate(progress * 100);
```

Comment:
I did it that way because the iOS version method takes a value between 0 and 1. Also, I saw the onProgressUpdate callback calls and seems to be receiving values between 0-100 so I assumed like a % value. I tested it in the sample and seems to work fine. Did I miss something?